portfolio/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Just some typescript config that you can ignore for now. Feel free to look into what each of these does. But that's pretty advanced stuff that definitely wouldn't be expected of an entry level dev to know